docs/user/dashboard/lens.asciidoc

```diff
@@ -319,7 +319,12 @@ You can also change the index pattern for a single *Layer*.
 [[why-my-field-x-is-missing-from-the-fields-list]]
 ===== Why is my field X missing from the fields list?
 
-*Lens* does not support the visualization of full-text fields, therefore it is not showing them in the data summary.
+If your field is not in the `Available fields` list in *Lens* there may be multiple reasons, here described:
+* the field is a full-text field, therefore it is not showing in the data summary.
```

Comment:
```suggestion
* The field is a full-text field, therefore it is not showing in the data summary.
```

Reply:
Instead of `therefore it is not showing in the data summary`, we should include why full-text fields do not appear.